package.json

```diff
@@ -68,7 +69,8 @@
     "taskcluster-lib-urls": "12.0.0",
     "victory": "33.0.5",
     "webpack": "4.39.3",
-    "webpack-cli": "3.3.8"
+    "webpack-cli": "3.3.8",
+    "yarn": "1.19.1"
```

Comment:
The only thing that should have changed is the addition of react-helmut, so you'll need to fix this file and the yarn.lock file. To resolve this you'll need to rebase onto the latest master - don't pull and merge. When you push your branch here do a force push with the `--force` flag.

Reply:
I can do that. 

I noticed that this project on master used yarn to install and start so on my fork and branch, it was the latest yarn so I updated it because it gave me errors. I’m not sure what your thoughts were on this. I just read errors and proceeded to follow it.

Reply:
Was this what you saw?
<img width="1408" alt="Screen Shot 2019-10-09 at 10 03 58 AM" src="https://user-images.githubusercontent.com/19615783/66504407-e1b41d80-ea7d-11e9-8ae1-b2c8fab48dcf.png">
When you upgrade your local version of yarn, you should be sure to only update your system and not the code base files. Running `brew upgrade yarn` has the desired effect. If you used `yarn` to update itself, that is likely why it also updated `package.json` and `yarn.lock` files. 

We use a bot to manage dependency upgrades so that's why we don't want any upgrades done as part of an unrelated pull request.

Reply:
I see merge conflicts now, so you'll need to rebase on master and resolve the conflicts. Be sure to run `yarn` to catch any changes to the `package.json` file.

Reply:
Sounds great! Thank you for clarifying. 

<img width="791" alt="Screen Shot 2019-10-09 at 2 55 13 PM" src="https://user-images.githubusercontent.com/38536770/66511452-f2c45500-eaa4-11e9-9d8b-8807bc0f35ad.png">

Reply:
That was the error message I got. It said to `yarn add react-helmet`

Reply:
Right, so you _do_ want to add `yarn add react-helmut`, you _do not_ want to update yarn with `yarn add yarn` - only via something like homebrew. I'm also seeing a lot of other packages updated in the lockfile.... did you also run `yarn upgrade`?

Reply:
I think the easiest way to clean this up is to do a soft reset of your commit and remove/discard all files you had commited except for the bugdetails view, see https://stackoverflow.com/questions/12481639/remove-files-from-git-commit

Then do `yarn add react-helmut` and commit the changes.

Don't do a force push to his branch unless you know that was successful, otherwise it'll override your work. You can make a copy of this branch first as a safeguard.

Reply:
Thank you for guiding me. Yeah I did do `yarn add yarn` but I should've read more into what that actually does instead. I'll make a new branch and do a soft reset.